scipy/special/Faddeeva.cc

```diff
@@ -125,7 +125,7 @@ extern "C" {
 #include <cfloat>
 #include <cmath>
 
-using namespace std;
+#define complex std::complex
```

Comment:
@tyhone confirmed in https://github.com/scipy/scipy/issues/5607#issuecomment-168213052 that this works.